### DISCUSSION TITLE
Scanner support anyone?

### DISCUSSION THREAD

**Discussion by ppgrainbow** [86Box/86Box#4405] [Ideas]
I would like to see handheld or flatbed scanner emulation, something which can be connected to a SCSI port.

For instance, the original Logitech ScanMan, introduced in 1989 was a SCSI device, there was a SCSI ID switch on the back. Here's a image of what it looks like: https://appletothecore.me/files/logitech_scanman.php

I'm wondering how this can be implemented. If you got any ideas, feel free to discuss. ^_^

**Comment by Sunspark-007** [comment]
Wow, I remember that device. It's been a lot of years since I've thought of it.

https://docs.mamedev.org/techspecs/nscsi.html

**Reply by ppgrainbow** [reply]
I agree. It would be awesome to implement Logitech ScanMan support!

So...does anyone have any idea how to implement handheld scanner support and to get it to work under 86Box?

**Comment by upintheairsheep** [comment]
More recent scanners should be emulated as well though.

**Reply by ppgrainbow** [reply]
I'm with you on this one. What I would like to focus mostly for the time being is the Logitech ScanMan series, most notably the ScanMan 256. Documentation will be need to see how the ScanMan would work when emulated in future releases of 86Box. :)
